README.md

```diff
@@ -332,7 +332,8 @@ changes in [UPGRADE.md](./UPGRADE.md) and [CHANGELOG.md](./CHANGELOG.md).
 
 ### Command line documentation
 
-Run `kratos -h` or `kratos help`.
+Run <code type="shell/command">kratos -h</code> or
+<code type="shell/command">kratos help</code>.
```

Comment:
Here you describe shell commands to get command-line help. I tag them as shell commands. Text-Runner executes shell commands in a subshell and verifies exit code 0. 

If somebody (hypothetically) changes the Kratos source so that the CLI argument is `kratos --help`, Text-Runner will enforce that this documentation gets updated as well!

These shell commands only succeed if somebody runs `make install` first to create the Kratos binary. This could be done by other tests (end-to-end tests?) that run before these documentation tests. Text-Runner can of course also run `make install`. It is currently documented below and takes a while, so I'm not doing it.